plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestTransactionLogAccess.java

```diff
@@ -736,6 +738,45 @@ public void testTableSnapshotsCacheDisabled()
                         "00000000000000000014.json", 2));
     }
 
+    @Test
+    public void testTableSnapshotsActiveDataFilesCacheExpiration()
+            throws Exception
+    {
+        String tableName = "person";
+        Path tableDir = new Path(getClass().getClassLoader().getResource("databricks/" + tableName).toURI());
+        DeltaLakeConfig shortLivedActiveDataFilesCacheConfig = new DeltaLakeConfig();
+        shortLivedActiveDataFilesCacheConfig.setDataFileCacheTtl(new Duration(10, TimeUnit.MILLISECONDS));
+        setupTransactionLogAccess(tableName, tableDir, shortLivedActiveDataFilesCacheConfig);
+
+        List<AddFileEntry> addFileEntries = transactionLogAccess.getActiveFiles(tableSnapshot, SESSION);
+        assertEquals(addFileEntries.size(), 12);
+        assertEquals(
+                hdfsEnvironment.getAccessedPathNames(),
+                ImmutableMap.of(
+                        "person", 2,
+                        "00000000000000000010.checkpoint.parquet", 2));
+
+        addFileEntries = transactionLogAccess.getActiveFiles(tableSnapshot, SESSION);
+        assertEquals(addFileEntries.size(), 12);
+        // The internal data cache should still contain the data files for the table
+        assertEquals(
```

Comment:
I have concern about the assumption of 10ms window. 2/1000 failed in my laptop.

Reply:
Increased the sleep time.

Reply:
Sorry, my explanation was unclear. The failure happened here, not after `sleepUninterruptibly`. If it takes more than 10ms between the 1st and 2nd `getActiveFiles()`, this test will fail. I know its' rare case though.

How about splitting into two test method? Default (or long duration) and disabled tests.